vault/resource_database_secret_backend_connection.go

```diff
@@ -1246,6 +1246,10 @@ func databaseSecretBackendConnectionRead(d *schema.ResourceData, meta interface{
 		d.Set("elasticsearch", getElasticsearchConnectionDetailsFromResponse(d, "elasticsearch.0.", resp))
 	case "snowflake-database-plugin":
 		d.Set("snowflake", getSnowflakeConnectionDetailsFromResponse(d, "snowflake.0.", resp))
+	case "redshift-database-plugin":
```

Comment:
Does this change warrant updating the related acceptance tests?

Reply:
I'll add a second step in the Redshift test to confirm updates and drift detection is working as expected

Reply:
@benashz added an update test in 7986b84